templates/helm.yml

```diff
@@ -122,6 +141,13 @@ deploy:quality:helm:
     name: ${ENVIRONMENT_NAME}
     url: https://${DOMAIN}
 
+deploy:quality:helm:branches:
+  extends: .helm:deploy
+  only:
+    - merge_requests
+  environment:
+    name: ${ENVIRONMENT_NAME}
+    url: https://${DOMAIN}
```

Comment:
If you add this here, it will automatically delete the environment.

```
    on_stop: .stop
```

With the only `but` here: `.stop` is a template, so I think it will not work directly this way. But the idea is right.

Reply:
Added that, me and @linuxbandit will discuss it later today and see if we can merge. Thanks!